tests/functional/sync_v3_sign_in.js

```diff
@@ -73,82 +75,179 @@ registerSuite('Firefox Desktop Sync v3 signin', {
   },
   tests: {
+    'open directly to /signin page, refresh on the /signin page': function () {
+      return this.remote
+        .then(createUser(email, PASSWORD, {preVerified: true}))
+        // redirected immediately to the / page
+        .then(openPage(SIGNIN_PAGE_URL, selectors.ENTER_EMAIL.HEADER, {
+          webChannelResponses: {
+            'fxaccounts:can_link_account': {ok: true}
+          }
+        }))
+        .then(type(selectors.ENTER_EMAIL.EMAIL, email))
+        .then(click(selectors.ENTER_EMAIL.SUBMIT, selectors.SIGNIN_PASSWORD.HEADER))
+
+        .refresh()
+
+        // refresh sends the user back to the first step
+        .then(testElementExists(selectors.ENTER_EMAIL.HEADER));
+    },
+
+    'user mistypes email': function () {
+      const forceUA = uaStrings['desktop_firefox_58'];
+      const query = {forceUA};
+      return this.remote
+        .then(createUser(email, PASSWORD, { preVerified: true }))
+        .then(openPage(ENTER_EMAIL_PAGE_URL, selectors.ENTER_EMAIL.HEADER, {
+          query,
+          webChannelResponses: {
+            'fxaccounts:can_link_account': { ok: true },
+            'fxaccounts:fxa_status': { capabilities: null, signedInUser: null },
+          }
+        }))
+        .then(visibleByQSA(selectors.ENTER_EMAIL.SUB_HEADER))
+        .then(type(selectors.ENTER_EMAIL.EMAIL, email))
+        .then(click(selectors.ENTER_EMAIL.SUBMIT, selectors.SIGNIN_PASSWORD.HEADER))
+        .then(testIsBrowserNotified('fxaccounts:can_link_account'))
+
+        // user thinks they mistyped their email
+        .then(click(selectors.SIGNIN_PASSWORD.LINK_MISTYPED_EMAIL, selectors.ENTER_EMAIL.HEADER))
+        .then(testElementValueEquals(selectors.ENTER_EMAIL.EMAIL, email))
+        .then(click(selectors.ENTER_EMAIL.SUBMIT, selectors.SIGNIN_PASSWORD.HEADER))
+
+        .then(testElementValueEquals(selectors.SIGNIN_PASSWORD.EMAIL, email));
+    },
+
     'Fx >= 58, verified, does not need to confirm ': function () {
       const forceUA = uaStrings['desktop_firefox_58'];
-      const query = {  forceUA };
+      const query = {forceUA};
 
       email = TestHelpers.createEmail();
 
       return this.remote
         .then(createUser(email, PASSWORD, {preVerified: true}))
-        .then(openPage(PAGE_URL, selectors.SIGNIN.HEADER, {
+        .then(openPage(ENTER_EMAIL_PAGE_URL, selectors.ENTER_EMAIL.HEADER, {
           query, webChannelResponses: {
             'fxaccounts:can_link_account': {ok: true},
             'fxaccounts:fxa_status': {capabilities: null, signedInUser: null},
           }
         }))
-        .then(fillOutSignIn(email, PASSWORD))
+        .then(fillOutEmailFirstSignIn(email, PASSWORD))
 
         .then(testElementExists(selectors.CONNECT_ANOTHER_DEVICE.HEADER));
     },
 
-    'verified, verify same browser, new tab\'s P.O.V ': function () {
-      return this.remote.then(setupTest({ preVerified: true}))
+    'Fx <= 57, verified, verify same browser': function () {
```

Comment:
I wonder how many users are using Fx <= 57. The current ESR is 60, so maybe we can drop tests that test below that?

Reply:
Our "official" policy is to support ESR - 1, and we still have a lot of tests hanging around for fx_desktop_v1 because we haven't officially deprecated it yet. I'll see if there are any that we can go "meh".